src/Core/OrganizationFeatures/OrganizationCollections/DeleteCollectionCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.OrganizationFeatures.OrganizationCollections.Interfaces;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+
+namespace Bit.Core.OrganizationFeatures.OrganizationCollections;
+
+public class DeleteCollectionCommand : IDeleteCollectionCommand
+{
+    private readonly ICollectionRepository _collectionRepository;
+    private readonly IEventService _eventService;
+    public DeleteCollectionCommand(
+        ICollectionRepository collectionRepository,
+        IEventService eventService)
+    {
+        _collectionRepository = collectionRepository;
+        _eventService = eventService;
+    }
+    public async Task DeleteAsync(Collection collection)
+    {
+        await _collectionRepository.DeleteAsync(collection);
+        await _eventService.LogCollectionEventAsync(collection, Enums.EventType.Collection_Deleted);
+    }
+    public async Task DeleteManyAsync(IEnumerable<Guid> collectionIds)
+    {
+
+        var ids = collectionIds as Guid[] ?? collectionIds.ToArray();
+        var collectionsToDelete = await _collectionRepository.GetManyByManyIdsAsync(ids);
+        await this.DeleteManyAsync(collectionsToDelete);
+
+    }
```

Comment:
nit: you're missing some whitespaces between methods and between constructor and class variables

Reply:
weird, would've thought `dotnet format` catches that...